lessons/javascript-fundamentals/functions-deep-dive/README.md

```diff
@@ -0,0 +1,61 @@
+Title | Tags
+--- | ---
+JavaScript Fundamentals: Workshop Review | javascript, arrays, conditionals, loops, functions
+
+## Objectives
+* Learn how arrays work under the hood
+* Be able to confidently answer all wkshp assessment questions (ex: find second largest element in an input array)
```

Comment:
The review portion is a great opportunity for student-led teaching.
Eg. maybe you can look thru the final assessments and ask students who got questions right to prepare demonstrations of their code. Another option would be to group up students of different levels to review the questions. 
The review is also a great opportunity to talk about HOW we learn -- to stress the importance of returning to topics multiple times and REDOING challenges.